src/test/java/jnr/posix/FileStatTest.java

```diff
@@ -157,4 +155,12 @@ public void filestatFailed() throws Throwable {
             }
         }
     }
+
+    @Test public void statIsWritable() throws Throwable {
+      String tempDirectory = System.getenv("TEMP");
+      FileStat st = posix.stat(tempDirectory);
+
+      // FIXME: This fails on JRuby, but succeeds here. Why?
```

Comment:
Having dug up the previous issue, I don't think this is related. But here's an example of Ruby applying additional logic:

https://github.com/jruby/jruby/blob/a968fc7db36d49ca3e09201f788bce7285d7231e/lib/ruby/stdlib/tmpdir.rb#L30-L38